.changelog/4981.feature.md

```diff
@@ -0,0 +1,27 @@
+go/worker/common/p2p: Bootstrap libp2p peers
+
+Seed nodes can now bootstrap libp2p peers. Bootstrapping can be enabled or
+disabled using a new configuration setting named
+`worker.p2p.discovery.bootstrap.enable`
+which can be used on seed nodes as well as non-seed nodes (e.g. clients,
+key managers).
+
+To run a seed node with custom configuration that supports bootstrapping
+one needs to configure the libp2p host first:
+
+- `worker.p2p.port` # host
```

Comment:
Should we move this directly under `p2p` at some point?

Reply:
Of course. Move most of the stuff to `p2p`, keep worker related things there, move registry discovery under discoveries, ...